app/src/main/res/drawable/indoor_floor_button.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle" >
+
+    <solid android:color="#ffffff" />
+
+    <stroke
+        android:width="1dp"
+        android:color="#791B0E" />
```

Comment:
Same, is this a new colour? If so then add it to the styles next to our primary and accent colours, if not then use one of the predefined colours..